### ISSUE TITLE
Trouble with toggle not working

### ISSUE THREAD

**Issue by jnichols300** [ga-wdi-exercises/project3#82]
The toggle is not working correctly, or at all for that matter and I can't seem to figure out why

**Comment by RobertAKARobin** [comment]
I need to see some code, and ideally an error message too!

**Comment by jnichols300** [comment]
I don't get an error message and the consolelog works as well. 

``` js
var LocationView = function(location) {
  this.loc = location;
  this.$el = $("<div class='location loc-style'></div>");
};

LocationView.prototype = {
  render: function(){
    var self = this;

    var showLocations = self.$el.find("");
    var locationsDiv = self.$el.find("div.location");

    this.$el.on("click", function(){
      console.log("setup click on click");
        self.toggleLocations(locationsDiv);
      });

    self.$el.append("<h3>" + self.loc.name + "</h3>");
    self.$el.append("<h4>" + self.loc.count + "</h4>");
    self.$el.append('<input type="button" value="upvote" class="upvote">');
    self.$el.append('<input type="button" value="downvote" class="downvote">');

    self.click();
    $('.loc-container').append(self.$el);
  },
  click: function(){
    var self = this;
       self.$el.find('h3').on('click', function(){
         console.log(self.loc);
       });
       self.$el.find('.upvote').on('click', function(){
         self.loc.postVote(true);
       });
       self.$el.find('.downvote').on('click', function(){
         self.loc.postVote(self.loc.id, false);
       });
     },
   toggleDiv: function(locationsDiv){
     console.log("test");
     if(this.$el.is(":visible")){
       console.log("Hide/Show");
       locationsDiv.siblings("div.location").text("Hide Details");
     } else {
       console.log("second statement");
       locationsDiv.siblings("div.location").text("Show Details");
     }
   },
  toggleLocations: function(locationsDiv){
    console.log("Locations Div on Click");
    var self = this;
    if(locationsDiv.children().length === 0){
        this.$el.append('<div><img src="' + this.loc.icon + '" height="32px" width="32px"></div>');
        this.$el.append("<div class='location-toggle'>" + "<span class='list-items'>Address: </span>" + this.loc.address + "</div>");
        this.$el.append("<div class='location-toggle'>"  + "<span class='list-items'>Types: </span>" + this.loc.types + "</div>");
        this.$el.append("<div class='location-toggle'>"  + "<span class='list-items'>Rating: </span>" + this.loc.rating + "</div>");
    }
    console.log("sup bruh");
    locationsDiv.toggle();
    this.toggleDiv(locationsDiv);
  },

};
```

**Comment by RobertAKARobin** [comment]
Which toggle? ToggleDiv or ToggleLocation?

**Comment by jnichols300** [comment]
The actual toggle locationsDiv.toggle();

**Comment by RobertAKARobin** [comment]
`var locationsDiv = self.$el.find("div.location");` is a local variable that exists only inside the `render()` function. If you want to access `locationsDiv` in the `toggleLocations()` function, you'll need to re-declare that variable inside the `toggleLocations()` function.

**Comment by jnichols300** [comment]
Hmmm I guess I didn't explain it very well sorry. So I got it to work, but it's referencing everything that's being created in that instance including the upvotes/downvotes etc. So when I toggle, it hides everything. Is there a way I can use toggle to reference the classes of that instance so I can hide what I created in toggleLocations? I've tried a ton of things and they aren't really working. Seems straight forward but I can't figure it out
